examples/material_gallery/lib/demo/flexible_space_demo.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class _ContactCategory extends StatelessComponent {
+  _ContactCategory({ this.icon, this.children });
```

Comment:
All these components should have key arguments in their constructors to show best practice.